### DISCUSSION TITLE
DT/NC on mania has 1.10x mod multipliers

### DISCUSSION THREAD

**Discussion by enoslayd** [ppy/osu#25939] [Ideas] [answered]
### Type

Game behaviour

### Bug description

DT/NC's multiplier should stay as 1.0x like in stable 

### Screenshots or videos

![osu_2023-12-20_14-48-05](https://github.com/ppy/osu/assets/41866328/4e68794f-e7f0-4ea0-bf76-dfce5219d6ab)

### Version

2023.1220.0

### Logs

[logs.zip](https://github.com/ppy/osu/files/13724018/logs.zip)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/25940
